### PR TITLE
docs: add comments and rename scripts

### PR DESCRIPTION
**Pull request by klesh**
# Summary

part of #3459

### Does this close any open issues?
related to #3459

### CODE REVIEW

File: models/migrationscripts/20220929_change_leadtimeminutes_to_int64.go [hunk 1]
```diff
@@ -35,7 +35,9 @@ func (Issues20220929) TableName() string {
 	return "issues"
 }
 
-func (*modifyLeadTimeMinutes) Up(basicRes core.BasicRes) errors.Error {
+func (*changeLeadTimeMinutesToInt64) Up(basicRes core.BasicRes) errors.Error {
+	// Yes, issues.lead_time_minutes might be negative, we ought to change the type
+	// for the column from `uint` to `int64`
```
Comment (likyh) [inline models/migrationscripts/20220929_change_leadtimeminutes_to_int64.go:40]: add issue link here?

Comment (klesh) [reply models/migrationscripts/20220929_change_leadtimeminutes_to_int64.go:40]: let me try try

**Review by mappjzc — APPROVED** [review]

**Review by abeizn — APPROVED** [review]

File: models/migrationscripts/20220929_change_leadtimeminutes_to_int64.go (hunk 1 above)
Comment (klesh) [reply models/migrationscripts/20220929_change_leadtimeminutes_to_int64.go:40]: done

**Review by likyh — APPROVED** [review]
LGTM

**Pull request merged**